servatrice/migrations/servatrice_0020_to_0021.sql

```diff
@@ -0,0 +1,12 @@
+
+
+CREATE TABLE IF NOT EXISTS `cockatrice_forgot_password` (
+  `id` int(7) unsigned zerofill NOT NULL auto_increment,
+  `name` varchar(35) NOT NULL,
+  `requestDate` datetime NOT NULL default '0000-00-00 00:00:00',
```

Comment:
The audit table can contain that information at a later time. I also agree that not having a default here could be troublesome. This default timestamp is fine.